talks/jacob-larfors.md

```diff
@@ -0,0 +1,70 @@
+# DevOps Malmö Talk Request
+
+By adding account details and/or Personal information, You give permission to share your information to the public repository.
+Including sharing on social media such as Meetup.com, Twitter, Linkedin, etc.
+> Note! Please don't share any Email addresses.
+
+## Speaker Info
+
+**Your Name:** Jacob Lärfors
+
+**Twitter (optional):** @jlarfors
+
+**Linkedin (optional):** @jlarfors
+
+**Github (optional):** @jlarfors
+
+## About Talk
+
+**Meetup Group**: DevOps Malmö
+
+**Date**: May 2022 (to be confirmed)
```

Comment:
Hi @jlarfors ,

do you have an exact date in your mind ? or waiting for confirmation from other parties?

Reply:
Yo @jlarfors let's do Wednesday the 4th :)

I would suggest adding this into `talks/jacob-larfors.md`  and in Readme file

as soon you have it ready will start organizing it :pray:

Reply:
Awesome :tada: looking forward to it. :handshake: